cernopendata_client/printer.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# cernopendata-client is free software; you can redistribute it and/or modify
+# it under the terms of the GPLv3 license; see LICENSE file for more details.
+
+"""cernopendata-client output print configuration."""
+
+import click
+
+
+def display_message(prefix=None, msg_type=None, msg=None):
+    """Display message in a similar style as run_command().
+
+    :param prefix: prefix to be added in output (==> / ->)
+    :param msg_type: Type of message - Error/Progress/Success
+    :param msg: message to display
+    :type prefix: str
+    :type msg_type: str
+    :type msg: str
+    """
+    msg_color_map = {
+        "success": "bright_green",
+        "progress": "bright_yellow",
+        "error": "red",
+    }
+    if prefix:
```

Comment:
My principal comment is that the leading prefix `==>` is printed in different colour than the rest of the headline, for example:

![2020-10-19-095745_586x478_scrot](https://user-images.githubusercontent.com/517546/96417414-8e78bd00-11f1-11eb-80b0-452a656411fa.png)

- in case of errors, the prefix `==>` should be red, to better visually indicate the nature of the report
- in case of info messages, ditto, the prefix `==>` should be in yellow
- the secondary prefix `->` is less important, so it can always stay in green; only the leading prefix should be coloured depending on the info vs error content

Reply:
(also, to use leading green colour for `==>` is nice, but the leading headline should then continue, e.g. simply in regular colour but in **bold**, or else continue with the green)

Reply:
BTW the same for success:

Reply:
Done 👍

Reply:
Actually I see this:

![2020-10-19-140222_582x130_scrot](https://user-images.githubusercontent.com/517546/96448472-baa53580-1213-11eb-861f-be928c996b04.png)

where the final Success is printed in the same colour as regular text... so it is less graphically apparent than the other Downloading/Verifying headlines, while it should be at least equally well represented.

Note also that the "-> " in Progress is not printed in bold as "-> " in Files.

Reply:
Now, `Success!` is equally apparent to other headings.

`->` is now same everywhere (not bold).

Reply:
Done 👍